modules/setting/setting.go

```diff
@@ -388,6 +387,11 @@ var (
 	UILocation = time.Local
 )
 
+// IsProd if it's a production mode
+func IsProd() bool {
+	return RunMode == "prod"
```

Comment:
In case someone sets it to "Prod", "PROD", etc?
```suggestion
	return strings.EqualFold(RunMode, "prod")
```

Reply:
Done.